projects/ad7606x_fmc/Readme.md

```diff
@@ -1,46 +1,36 @@
-  * [EVAL-AD7606C-16/18 Product Page](https://www.analog.com/en/design-center/evaluation-hardware-and-software/evaluation-boards-kits/eval-ad7606c-18.html)
-  * [EVAL-AD7605-4 Product Page](https://www.analog.com/en/design-center/evaluation-hardware-and-software/evaluation-boards-kits/eval-ad7605-4.html)
-  * [EVAL-AD7606-8/6/4 Product Page](https://www.analog.com/en/design-center/evaluation-hardware-and-software/evaluation-boards-kits/eval-ad7606.html)
-  * [EVAL-AD7607 Product Page](https://www.analog.com/en/resources/evaluation-hardware-and-software/evaluation-boards-kits/eval-ad7607.html)
-  * [EVAL-AD7608 Product Page](https://www.analog.com/en/resources/evaluation-hardware-and-software/evaluation-boards-kits/eval-ad7608.html)
-  * [EVAL-AD7609 Product Page](https://www.analog.com/en/resources/evaluation-hardware-and-software/evaluation-boards-kits/eval-ad7609.html)
-  * Parts : [AD7606B, 8 Channels, 16-bit, 800 kSPS Bipolar Input, Simultaneous sampling ADC](https://www.analog.com/en/products/ad7606b.html)
-  * Parts : [AD7606C-16, 8 Channels, 16-bit, 1 MSPS Bipolar Input, Simultaneous sampling ADC](https://www.analog.com/en/products/ad7606c-16.html)
-  * Parts : [AD7606C-18, 8 Channels, 18-bit, 1 MSPS Bipolar Input, Simultaneous sampling ADC](https://www.analog.com/en/products/ad7606c-18.html)
-  * Parts : [AD7605-4, 4 Channels, 16-bit, 300 kSPS Bipolar Input, Simultaneous sampling ADC](https://www.analog.com/en/products/ad7605-4.html)
-  * Parts : [AD7606, 8 Channels, 16-bit, 200 kSPS Bipolar Input, Simultaneous sampling ADC](https://www.analog.com/en/products/ad7606.html)
-  * Parts : [AD7606-6, 6 Channels, 16-bit, 200 kSPS Bipolar Input, Simultaneous sampling ADC](https://www.analog.com/en/products/ad7606-6.html)
-  * Parts : [AD7606-4, 4 Channels, 16-bit, 200 kSPS Bipolar Input, Simultaneous sampling ADC](https://www.analog.com/en/products/ad7606-4.html)
-  * Parts : [AD7607, 8 Channels, 14-bit, 200 kSPS Bipolar Input, Simultaneous sampling ADC](https://www.analog.com/en/products/ad7607.html)
-  * Parts : [AD7608, 8 Channels, 18-bit, 200 kSPS Bipolar Input, Simultaneous sampling ADC](https://www.analog.com/en/products/ad7608.html)
-  * Parts : [AD7609, 8 Channels, 18-bit, 200 kSPS Bipolar Input, Simultaneous sampling ADC](https://www.analog.com/en/products/ad7609.html)
-  * Project Doc: https://wiki.analog.com/resources/eval/user-guides/ad7606x-fmcz
-  * HDL Doc: https://wiki.analog.com/resources/eval/user-guides/ad7606x-fmc/hdl
-  * NO-OS Drivers: [AD7606 - No-OS Driver](https://wiki.analog.com/resources/tools-software/uc-drivers/ad7606)
-  * Linux Drivers: https://wiki.analog.com/resources/tools-software/linux-drivers/iio-adc/axi-adc-hdl
-## Building, Generating Bit Files
+- Evaluation board product page:
+  * [EVAL-AD7606B](https://www.analog.com/en/design-center/evaluation-hardware-and-software/evaluation-boards-kits/eval-ad7606b-fmcz.html)
```

Comment:
Please use this format for the boards list: [example](https://github.com/analogdevicesinc/hdl/blob/main/projects/pulsar_lvds_adc/README.md)